log.go

```diff
@@ -71,6 +71,7 @@ var (
 	sgnrLog = build.NewSubLogger("SGNR", backendLog.Logger)
 	wlktLog = build.NewSubLogger("WLKT", backendLog.Logger)
 	arpcLog = build.NewSubLogger("ARPC", backendLog.Logger)
+	ntfrLog = build.NewSubLogger("NTFR", backendLog.Logger)
```

Comment:
`ntfrLog` needs to be added to the `subsystemLoggers` list too.

Reply:
Fixed.